src/components/loading-slates/loading-slates.tsx

```diff
@@ -27,7 +28,8 @@ export const SeekbarPlaceholder = () => (
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
     preserveAspectRatio="none"
-    className={[styles.animated, styles.seekbarPlaceholder].join(' ')}>
+    className={[styles.animated, styles.seekbarPlaceholder].join(' ')}
+  >
     <path
       opacity="0.3"
       d="M0.625 4.36572H3.095V5.12072H0.625V4.36572ZM5.13594 4.36572H7.60594V5.12072H5.13594V4.36572ZM9.64688 7.19072C9.64688 7.10406 9.66188 7.02239 9.69188 6.94572C9.72521 6.86906 9.76854 6.80239 9.82188 6.74572C9.87854 6.68906 9.94521 6.64406 10.0219 6.61072C10.0985 6.57739 10.1802 6.56072 10.2669 6.56072C10.3535 6.56072 10.4352 6.57739 10.5119 6.61072C10.5885 6.64406 10.6552 6.68906 10.7119 6.74572C10.7685 6.80239 10.8135 6.86906 10.8469 6.94572C10.8802 7.02239 10.8969 7.10406 10.8969 7.19072C10.8969 7.28072 10.8802 7.36406 10.8469 7.44072C10.8135 7.51406 10.7685 7.57906 10.7119 7.63572C10.6552 7.69239 10.5885 7.73572 10.5119 7.76572C10.4352 7.79906 10.3535 7.81572 10.2669 7.81572C10.1802 7.81572 10.0985 7.79906 10.0219 7.76572C9.94521 7.73572 9.87854 7.69239 9.82188 7.63572C9.76854 7.57906 9.72521 7.51406 9.69188 7.44072C9.66188 7.36406 9.64688 7.28072 9.64688 7.19072ZM9.64688 3.44072C9.64688 3.35406 9.66188 3.27239 9.69188 3.19572C9.72521 3.11906 9.76854 3.05239 9.82188 2.99572C9.87854 2.93906 9.94521 2.89406 10.0219 2.86072C10.0985 2.82739 10.1802 2.81072 10.2669 2.81072C10.3535 2.81072 10.4352 2.82739 10.5119 2.86072C10.5885 2.89406 10.6552 2.93906 10.7119 2.99572C10.7685 3.05239 10.8135 3.11906 10.8469 3.19572C10.8802 3.27239 10.8969 3.35406 10.8969 3.44072C10.8969 3.53072 10.8802 3.61406 10.8469 3.69072C10.8135 3.76406 10.7685 3.82906 10.7119 3.88572C10.6552 3.94239 10.5885 3.98739 10.5119 4.02072C10.4352 4.05072 10.3535 4.06572 10.2669 4.06572C10.1802 4.06572 10.0985 4.05072 10.0219 4.02072C9.94521 3.98739 9.87854 3.94239 9.82188 3.88572C9.76854 3.82906 9.72521 3.76406 9.69188 3.69072C9.66188 3.61406 9.64688 3.53072 9.64688 3.44072ZM12.9469 4.36572H15.4169V5.12072H12.9469V4.36572ZM17.4578 4.36572H19.9278V5.12072H17.4578V4.36572Z"
```

Comment:
i think this should be in a separate file

Reply:
i did not change anything here